instaloader/instaloadercontext.py

```diff
@@ -335,8 +335,8 @@ def get_json(self, path: str, params: Dict[str, Any], host: str = 'www.instagram
                     redirect_url.startswith('https://i.instagram.com/accounts/login')):
                     if not self.is_logged_in:
                         raise LoginRequiredException("Redirected to login page. Use --login.")
-                    # alternate rate limit exceeded behavior
-                    raise TooManyRequestsException("Redirected to login")
+                    raise AbortDownloadException("Redirected to login page. You've been logged, please wait " +
```

Comment:
```suggestion
                    raise AbortDownloadException("Redirected to login page. You've been logged out, please wait " +
```

Should read "You've been logged **out**", I guess?